backend/kale/marshal/__init__.py

```diff
@@ -19,5 +19,5 @@
 
 
 from kale.utils import log_utils
-log_utils.get_or_create_logger(module=__name__, name="Kale Marshalling")
+log_utils.get_or_create_logger(module=__name__, name="marshaling")
```

Comment:
Can we call it `marshalling`, with two`l`s? I searched and it looks like both are valid, but the double `l` version seems to be the one more common

Reply:
Fixed!